content/academy/become_an_apify_expert.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: Pro Apify scraper development
+title: Become an Apify expert
```

Comment:
Maybe this title should be `Professional scraping`? `become an Apify expert` suggests to me that the course is a continuation of the `Apify platform` course

Reply:
@m-murasovs I had it as **Professional scraping**, but then @metalwarrior665 said it should be something more like **Become Apify expert**

Reply:
It is a bit of a continuation of the **Apify platform** course and the **Web scraping for beginner** course.

Reply:
cc @metalwarrior665 && @mnmkng  Thoughts?

Reply:
My latest suggestion was to ask inside the company, I don't want to make a singular decision about this.

Reply:
I will ask for feedback in #proj-academy